tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,182 @@
+Feature: Build and Run commands for CapsulaHub CLI
+
+  Background:
+    Given an npm package called @capsulajs/capsula-hub
+    And   a default port 55555
+    And   a default configProvider httpFile
+    And   the following <configProvider> available
+          |<configProvider>                   |
+          |LocalFileConfigurationProvider     |
+          |HttpFileConfigurationProvider      |
+          |HardcoreServerConfigurationProvider|
+          |HttpServerConfigurationProvider    |
+          |LocalStorageConfigurationProvider  |
+    And   a default path to the output folder "./dist"
+    And   a valid port is a number between 1024 and 65535
+
+  Scenario: Run Capsula-Hub instance with specifying valid arguments
+    Given a valid token with configuration
+    When  I run the command `capsula-hub run --token="token" --configProvider="configProvider" --dispatcherUrl="dispatcherUrl" --port="port"` #add dispatcherUrl only for hardcoreServer provider
+    Then  an workspace is created
+    And   workspace is running on localhost on the specifying port
+    And   the specified configProvider is used
+
+  Scenario: Run Capsula-Hub instance without specifying the port and configProvider
+    Given a valid token with configuration
+    When  I run the command `capsula-hub run --token="token"`
+    Then  an workspace is created
+    And   workspace is running on localhost on the default port
+    And   httpFile is the configuration provider used
+
+  Scenario: Run Capsula-Hub instance with invalid token
+    When  I run the command `capsula-hub run --token="token"`
+    And   one of the following values for <token> is provided
+          |<token>     |
+          |''        |
+          |{}        |
+          |{ test: 'test' }|
+          |[]        |
+          |['test']  |
+          |null      |
+          |undefined |
+          |true      |
+          |false     |
+          |0         |
+          |-1        |
+    Then  a relevant error is received
+
+  Scenario: Run Capsula-Hub instance with invalid port
+    Given a valid token with configuration
+    When  I run the command `capsula-hub run --token="token" --port="port" `
+    And   one of the following values for <port> is provided
+          |<port>    |
+          | 'abc'    |
+          | 65536    |
+          | 0        |
+          |{}        |
+          |{ test: 'test' }|
+          |[]        |
+          |['test']  |
+          |null      |
+          |undefined |
+          |true      |
+          |false     |
+          |-1        |
+    Then  a relevant error is received
```

Comment:
I think it's not needed - args in a terminal are treated as strings anyway, so one value with smth like "invalidPort" is enough.